### ISSUE TITLE
kmozhelper crashes after clicking Save As

### ISSUE THREAD

**Issue by geck0en** [moz-plasma/firefox-plasma-support#18]
```
Thread 1 "firefox" received signal SIGPIPE, Broken pipe.
0x00007ffff6e3eced in write () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.
```
Apparmor report:
[_usr_lib_mozilla_kmozillahelper.1000.zip](https://github.com/plasmazilla/mozilla-kde-support/files/779634/_usr_lib_mozilla_kmozillahelper.1000.zip)

I'm going to investigate the issue further more and will come back with some more updates.

**Comment by shadeslayer** [comment]
@llelectronics was the plasmazilla PPA updated with the fix from upstream?

**Comment by llelectronics** [comment]
The fix is in the builds but not released yet. Need to test it first.
**Update: Tested and pushed to releases plasmazilla PPA. Fix should appear soon.**

**Comment by geck0en** [comment]
My apologies for not getting back sooner, I got caught up with other things.
I ran into a fix for unknown filetype (couldn't find it again so unable to link that in).

However the files I experienced the reported issue with were _BIN_ ones:
![image](https://cloud.githubusercontent.com/assets/25816022/23022687/05a19aba-f44a-11e6-9926-ec3fa3c854ed.png)

Based on @llelectronics reply I've altered the ppa repository from _releases_ to _builds_ and after the system update I was unable to replicate the crash.

Thank you all for your quick help and for all your work in the open software world.
I'm closing this call off.
